docs/releases/1.10.txt

```diff
@@ -97,7 +97,13 @@ Minor features
 :mod:`django.contrib.contenttypes`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* :class:`~django.contrib.contenttypes.fields.GenericForeignKey` no longer
```

Comment:
The note about the `pre_init` signal is probably not noteworthy, but the other one, about reloading GFK targets, is a change in behavior that might be considered a bugfix.